test/specs/WebexMeeting/MeetingPage.js

```diff
@@ -0,0 +1,79 @@
+import {createTestUser, removeTestUser} from '@webex/test-users';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+class MeetingPage {
+  constructor() {
+    this.user = null;
+  }
+
+  setUser(user) {
+    this.user = user;
+  }
+  // eslint-disable-next-line
+  get accessToken() { return $('#token'); }
+  // eslint-disable-next-line
+  get destination() { return $('#destination'); }
+  // eslint-disable-next-line
+  get saveTokenBtn() { return $('#md-button-1'); }
+  // eslint-disable-next-line
+  get displayWidgetBtn() { return $('#md-button-3'); }
+
+  // eslint-disable-next-line
+  get waitingForOthers() { return $('h4=Waiting for others to join...'); }
+  // eslint-disable-next-line
+  get interstitialMeeting() { return $('.wxc-interstitial-meeting'); }
+
+  open() {
+    browser.url(process.env.WEBEX_TEST_PAGE_URL);
+  }
+
+  loadWidget() {
+    this.open();
+    this.accessToken.setValue(process.env.WEBEX_ACCESS_TOKEN);
+    this.saveTokenBtn.click();
+    this.destination.setValue(process.env.WEBEX_MEETING_DESTINATION);
+    this.displayWidgetBtn.click();
+
+    this.waitForInterstitialScreen();
+  }
+
+  createUser() {
```

Comment:
let's move the user creation/deletion out of the `Page` class, they don't seem to belong to a particular page.

You can create a separate module called, say, `util.js`, and then import the user functions in the spec file:

```js
import {createUser, removeUser} from './util';

describe('Meeting Widget', () => {
  let user;
  
  beforeAll(() => {
    user = await createUser();
  });

  afterAll(() => {
    removeUser(user);
  });
});
```